qa/L0_client_tar/test.sh

```diff
@@ -32,75 +32,42 @@ rm -fr triton_client
 mkdir triton_client
 (cd triton_client && tar xzvf /workspace/*.tar.gz)
 
-# Build
-cd triton_client/build
-cmake -DCMAKE_BUILD_TYPE=Release -DTRITON_CLIENT_CMAKE_DIR:PATH=`pwd`/../lib/cmake/TRITON .
-make -j16 client
+RET=0
 
-# There is no server running but can still check to make sure that the
-# example application starts correctly.
-set +e
-
-# Shared HTTP
-client/install/bin/simple_client_shared > $CLIENT_LOG 2>&1
-if [ $? -ne 1 ]; then
-    cat $CLIENT_LOG
-    echo -e "\n***\n*** Unexpected Pass for simple_client_shared HTTP\n***"
-    exit 1
-fi
-
-grep -c "Couldn't connect to server" $CLIENT_LOG
-if [ $? -ne 0 ]; then
-    cat $CLIENT_LOG
-    echo -e "\n***\n*** Test Failed\n***"
-    exit 1
+# Check image_client and perf_client
+if [[ ! -x "bin/image_client" ]]; then
+    echo -e "*** image_client executable not present\n"
+    RET=1
 fi
-
-# Shared GRPC
-client/install/bin/simple_client_shared -i grpc > $CLIENT_LOG 2>&1
-if [ $? -ne 1 ]; then
-    cat $CLIENT_LOG
-    echo -e "\n***\n*** Unexpected Pass for simple_client_shared GRPC\n***"
-    exit 1
+if [[ ! -x "bin/perf_client" ]]; then
+    echo -e "*** perf_client executable not present\n"
+    RET=1
 fi
 
-grep -c "failed to connect" $CLIENT_LOG
-if [ $? -ne 0 ]; then
-    cat $CLIENT_LOG
-    echo -e "\n***\n*** Test Failed\n***"
-    exit 1
-fi
+# Check static libraries
+for l in libgrpcclient.so libgrpcclient_static.a libhttpclient.so libhttpclient_static.a; do
```

Comment:
At some point, I think we must add the tests for these libraries than just checking there presence,